[2025-03-23T14:11:59.086+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-23T14:11:59.141+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: realtime_user_streaming_pipeline.extract_data manual__2025-03-23T14:11:56.655793+00:00 [queued]>
[2025-03-23T14:11:59.147+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: realtime_user_streaming_pipeline.extract_data manual__2025-03-23T14:11:56.655793+00:00 [queued]>
[2025-03-23T14:11:59.147+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2025-03-23T14:11:59.157+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_data> on 2025-03-23 14:11:56.655793+00:00
[2025-03-23T14:11:59.175+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=327) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-23T14:11:59.176+0000] {standard_task_runner.py:64} INFO - Started process 328 to run task
[2025-03-23T14:11:59.172+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'realtime_user_streaming_pipeline', 'extract_data', 'manual__2025-03-23T14:11:56.655793+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/realtime_streaming.py', '--cfg-path', '/tmp/tmpcg84wmz_']
[2025-03-23T14:11:59.183+0000] {standard_task_runner.py:91} INFO - Job 21: Subtask extract_data
[2025-03-23T14:11:59.419+0000] {task_command.py:426} INFO - Running <TaskInstance: realtime_user_streaming_pipeline.extract_data manual__2025-03-23T14:11:56.655793+00:00 [running]> on host 76911cf2b4bd
[2025-03-23T14:11:59.689+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='realtime_user_streaming_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-23T14:11:56.655793+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-23T14:11:56.655793+00:00'
[2025-03-23T14:11:59.695+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-23T14:12:01.815+0000] {etl_pipeline.py:17} INFO - Fetched 100 users successfully.
[2025-03-23T14:12:01.979+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-23T14:12:01.986+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-23T14:12:02.030+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=realtime_user_streaming_pipeline, task_id=extract_data, run_id=manual__2025-03-23T14:11:56.655793+00:00, execution_date=20250323T141156, start_date=20250323T141159, end_date=20250323T141202
[2025-03-23T14:12:02.229+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-23T14:12:02.288+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-23T14:12:02.293+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
